tests/catch/include/hip_test_common.hh

```diff
@@ -329,11 +329,89 @@ struct MemTraits<MemcpyAsync> {
   }
 };
 
-}  // namespace HipTest
 
+namespace {
+static __global__ void waitKernel(clock_t offset) {
```

Comment:
Changes to this file need to be removed before removing the draft attribute

Reply:
Done